ai/extract_features.py

```
@@ -114,18 +112,14 @@ def batch_process(texts, batch_size=10000):
     return results
 
 
-def _extracted_from_extract_features_76(
-    vectorizer, df, category, n_components, random_state
-):
+def _extracted_from_extract_features_76(vectorizer, df, category, n_components, random_state):
     """Process smaller datasets all at once"""
     # For smaller datasets, process all at once
     print("Transforming data with TF-IDF...")
```

Ruff (0.11.9)

`print` found

Remove `print`

(T201)